src/EnergyPlus/Construction.cc

```diff
@@ -506,10 +503,6 @@ void ConstructionProps::calculateTransferFunction(EnergyPlusData &state, bool &E
                         Nodes(Layer) = ipts1;
                     }
 
-                    if (this->SolutionDimensions > 1) {
-                        if (ipts1 > Construction::MaxCTFTerms / 2) ipts1 = Construction::MaxCTFTerms / 2;
-                    }
-
```

Comment:
The line number changed but this is the suggestion for this change.

    [src/EnergyPlus/Construction.cc:477]:(style),[unreadVariable],Variable 'ipts1' is assigned a value that is never used.

Reply:
OK, I had to expand the hidden lines to confirm.  Even though this is inside the loop, it's always reassigned above before used, so yeah this isn't needed. :+1: